systems_cross_building/windows_subsystems.rst

```diff
@@ -74,7 +74,7 @@ the right order.
 
 There are some packages you can use as ``build_requires``:
 
-- From Conan-center:
+- From conancenter:
 
     - **mingw_installer/1.0@conan/stable**: MinGW compiler installer as a Conan package.
```

Comment:
mingw_installer@conan/stable should be removed too

Reply:
```suggestion
    - **mingw-w64/8.1**: MinGW compiler installer as a Conan package.
```